specification/2.0/README.md

```diff
@@ -1250,6 +1250,8 @@ glTF 2.0 also supports animation of instantiated Morph Targets in a similar fash
 
 > **Note:** glTF 2.0 defines only animation storage, so this specification doesn't define any particular runtime behavior, such as: order of playing, auto-start, loops, mapping of timelines, etc...
 
+> **Note:** glTF 2.0 does not specifically define how an animation will be used when imported but, as a best practice, it is recommended that each animation is self contained as an action. This clarification is necessary due to early glTF 2.0 exporters splitting the tracks of a single source animation into several glTF animations (behaviour that is not currently advised today).
```

Comment:
A minor thing: we did a pass at some point to use the words "client implementation" instead of viewer or engine.